prepare_source_tarball.sh

```diff
@@ -14,30 +11,90 @@ function define_default_value()
 	[ -z "${!VAR_NAME}" ] && export ${VAR_NAME}="${DEFAULT_VALUE}"
 }
 
-define_default_value GIT_DIR "$(realpath $(dirname $BASH_SOURCE))/../../.git/"
 define_default_value WORK_DIR "$(mktemp --directory)/"
+define_default_value TAR_FILE "RetroShare.tar.gz"
+define_default_value SRC_DIR "$(realpath $(dirname $BASH_SOURCE)/../../)"
 
 ORIG_DIR="$(pwd)"
 
-[ "$(ls "${GIT_DIR}/../supportlibs/restbed/" | wc -l)" -lt "5" ] &&
+[ "$(ls "${SRC_DIR}/supportlibs/restbed/" | wc -l)" -lt "5" ] &&
```

Comment:
Now it is much better! The ideal would be to have this parts in an external script so we can call it also from qmake and avoid code duplication

Reply:
Yes in the(your) next release. ;)